packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -667,6 +667,28 @@ function processDebugValues(
   }
 }
 
+function handleRenderFunctionError(error: any): void {
+  // original error might be any type.
+  const isError = error instanceof Error;
+  if (isError && error.name === 'ReactDebugToolsUnsupportedFeatureError') {
```

Comment:
Nit: `if (error instanceof Error && error.name === 'ReactDebugToolsUnsupportedFeatureError')`

Reply:
`isError` is used again in the `message` part so I created a variable to avoid calling it the second time. But I guess I will switch to a static message, so sure.